### ISSUE TITLE
There is a way to set the font size?

### ISSUE THREAD

**Issue by emleonardelli** [skmail/responsive-viewer#59]
For simulate some users that change the font size of their phones

**Comment by skmail** [comment]
no, there's no way to set font size from responsive viewer, but can explain more on how this feature?
